courses/machine_learning/deepdive2/time_series_prediction/labs/1_data_exploration.ipynb

```diff
@@ -0,0 +1,1457 @@
+{
```

Comment:
<p>BigQuery to the heavy-duty -&gt; BigQuery do the heavy-duty</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>